spec/features/applications/no_ni_number_and_undetermined_result_spec.rb

```diff
@@ -0,0 +1,119 @@
+# coding: utf-8
+require 'rails_helper'
+
+def personal_details_without_ni_number
+  login_as user
+  visit applications_new_path
+
+  fill_in 'application_last_name', with: 'Smith'
+  fill_in 'application_date_of_birth', with: Time.zone.today - 25.years
+  choose 'application_married_false'
+  click_button 'Next'
+end
+
+def application_details
+  fill_in 'application_fee', with: 410
+  find(:xpath, '(//input[starts-with(@id,"application_jurisdiction_id_")])[1]').click
+  fill_in 'application_date_received', with: Time.zone.today
+  click_button 'Next'
+end
+
+def savings_and_investments
+  choose 'application_threshold_exceeded_false'
+  click_button 'Next'
+end
+
+def benefits_page
+  choose 'application_benefits_true'
+  click_button 'Next'
+end
+
+RSpec.feature 'No NI number provided', type: :feature do
+
+  include Warden::Test::Helpers
+  Warden.test_mode!
+
+  let!(:jurisdictions) { create_list :jurisdiction, 3 }
+  let!(:office)        { create(:office, jurisdictions: jurisdictions) }
+  let!(:user)          { create(:user, jurisdiction_id: jurisdictions[1].id, office: office) }
+  let(:paper_evidence) { 'The applicant has provided paper evidence' }
+  let(:no_remission)   { 'The applicant must pay the full fee' }
+
+  before do
+    personal_details_without_ni_number
+    application_details
+    savings_and_investments
+    benefits_page
+  end
+
+  scenario 'correct warning message' do
+    warning_string = "The applicant's details could not be checked with the Department for Work and Pensions"
+    expect(page).to have_content warning_string
+  end
+
+  scenario '"Next" button' do
```

Comment:
Looking from the angle that you're looking at it, good observation. But I had it in, as I was developing the feature by feature, going from the outside.

Fine, I'll look at taking this out.